server/src/test/java/org/elasticsearch/action/admin/indices/close/CloseIndexRequestTests.java

```diff
@@ -54,7 +52,7 @@ public void testBwcSerialization() throws Exception {
         {
             final CloseIndexRequest request = randomRequest();
             try (BytesStreamOutput out = new BytesStreamOutput()) {
-                out.setVersion(randomVersionBetween(random(), Version.V_6_4_0, VersionUtils.getPreviousVersion(Version.V_7_2_0)));
+                out.setVersion(VersionUtils.randomCompatibleVersion(random(), Version.CURRENT));
```

Comment:
Should this be random version on or before 7.2? Seems by doing compatible version we have increased the possible versions it could be

Reply:
uh oh. you're right